php-frontend/src/main/java/org/sonar/php/parser/PHPGrammar.java

```diff
@@ -1417,7 +1420,7 @@ public AssignmentExpressionTree ASSIGNMENT_BY_REFERENCE() {
 
   public AssignmentExpressionTree ASSIGNMENT_EXPRESSION() {
     return b.<AssignmentExpressionTree>nonterminal(PHPLexicalGrammar.ASSIGNMENT_EXPRESSION).is(b.firstOf(
-      f.assignmentExpression(MEMBER_EXPRESSION(), ASSIGNMENT_OPERATOR(), EXPRESSION()),
+      f.assignmentExpression(MEMBER_EXPRESSION(), ASSIGNMENT_OPERATOR(), b.firstOf(ASSIGNMENT_EXPRESSION(), CONDITIONAL_EXPR())),
```

Comment:
just realized that we don't need `ASSIGNMENT_EXPRESSION()` here

Reply:
indeed, well spotted